static/solo/js/menu.js

```diff
@@ -0,0 +1,169 @@
+;(function(window, document) {
+    var menuTemplate = '<nav id="tvp-hidden-menu"></nav><div id="tvp-hamburger-container"><div class="tvp-hamburger tvp-hamburger-x"><span>.</span></div></div>',
+
+        itemTemplate = '<div id="tvp-video-{id}" class="tvp-video{className}">' +
+                            '<div class="tvp-video-image" style="background-image:url({asset.thumbnailUrl})">' +
+                                '<div class="tvp-active-overlay"><p>Now Playing</p></div>' +
+                                '<svg class="tvp-video-play" viewBox="0 0 200 200" alt="Play video">' +
+                                    '<polygon points="70, 55 70, 145 145, 100"></polygon>' +
+                                '</svg>' +
+                            '</div>' +
+                            '<div class="tvp-video-details">' +
+                                '<p class="tvp-video-title">{title}</p>' +
+                                '<p class="tvp-video-duration">-{asset.prettyDuration}</p>' +
+                            '</div>' +
+                        '</div>',
+
+    isset = function(o, p) {
+      var val = o;
+      if (p) val = o[p];
+      return 'undefined' !== typeof val;
+    },
+
+    isEmpty = function(obj) {
```

Comment:
don't we have this on Utils? it's prpbably cleaner to move this generic helpers there:

isset, isEmpty, tmpl, trimText, etc.